nncf/quantization/algorithms/weight_compression/weight_lowering.py

```diff
@@ -155,6 +155,31 @@ def calculate_e2m1_scale(weight: Tensor, reduction_axes: ReductionAxes, max_val=
     return scale
 
 
+def calculate_signed_scale(weight: Tensor, reduction_axes: ReductionAxes, num_bits=4) -> Tensor:
+    """
+    Calculates the scale for e2m1 quantization.
```

Comment:
Is this message for "e2m1" relevant here?

Reply:
Done